src/foam/u2/CurrencyView.js

```diff
@@ -0,0 +1,80 @@
+/**
```

Comment:
User shorter contributors copyright header. Set year to 2017.

/**
 * @license
 * Copyright 2017 The FOAM Authors. All Rights Reserved.
 * http://www.apache.org/licenses/LICENSE-2.0
 */